Assets/Scripts/AudioManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine.Audio;
+using System;
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager instance;
+    public string sceneMusic;
+    [SerializeField]
+    public Sound[] sounds;
+
+    void Awake()
+    {
+        if (instance != null)
+        {
+            Destroy(gameObject);
+        }
+        else
+        {
+            instance = this;
+            DontDestroyOnLoad(gameObject);
+        }
+
+        foreach (Sound sound in sounds)
+        {
+            sound.source = gameObject.AddComponent<AudioSource>();
+            sound.source.clip = sound.clip;
+            sound.source.loop = sound.loop;
+            sound.source.outputAudioMixerGroup = sound.mixerGroup;
+        }
+    }
+
+    void Start()
+    {
+        PlaySound(sceneMusic);
+    }
+
+    public void PlaySound(string _name)
```

Comment:
Could we decentralize the sounds? The current implementation requires us to manually keep the sound name strings in sync everywhere. Instead, could we have individual objects pass in the sound they want to play?

Something like this:

public void PlaySound(Sound _sound)
{
    if (_sound)
        _sound.PlaySound();
}

Note: I think it makes sense to keep background music centralized though, so we can change music like you do below